src/appengine/handlers/cron/project_setup.py

```diff
@@ -790,11 +790,14 @@ def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
             backup_bucket=backup_bucket_name)
 
       if self._add_info_labels:
-        job.environment_string += (
-            'AUTOMATIC_LABELS = Proj-{project},Engine-{engine}\n'.format(
-                project=project,
-                engine=template.engine,
-            ))
+        automatic_labels = 'Proj-{project},Engine-{engine}'.format(
+            project=project,
+            engine=template.engine,
+        )
+        labels = info.get('labels')
+        if labels and '*' in labels:
+          automatic_labels += ',%s' % ','.join(labels['*'])
+        job.environment_string += ('AUTOMATIC_LABELS = %s\n' % automatic_labels)
```

Comment:
nit: can we clean this up a bit by doing:

```python
automatic_labels = [
  f 'Project-{project}',
  f 'Engine-{engine}',
]

labels = info.get('labels')
if labels and '*' in labels:
  automatic_labels.extend(labels['*'])

automatic_labels = ','.join(automatic_labels)
job.environment_string += f'AUTOMATIC_LABELS = {automatic_labels}\n'
```
```

Reply:
nice, done! thanks